ajax/taskjob_logs.php

```diff
@@ -46,7 +46,6 @@
 }
 
 //unlock session since access checks have been done
-session_write_close();
```

Comment:
no, in this big commit somewhere, we have a big optimisation for data loading and mustache loop.

as far i remember, we had a significant improvement of loading (more than 50x) with a big load of data